kubernetes/envoy.yaml

```diff
@@ -0,0 +1,82 @@
+    admin:
```

Comment:
@lackrobin can you implement this on GCloud alone or do we need to adapt the source code?

Reply:
I've been trying to do this for a bit, but I have not found a good solution. I think it would be best to add a envoy.yaml file with our config to the orchestra repo, so we can copy it from there, and we also can edit it there accordingly should the original envoy.yaml file change.

Reply:
https://stackoverflow.com/questions/54047568/how-can-i-use-environment-variables-in-the-envoyproxy-config-file#54112029

This would be another strategy, but would also require some refactoring in the base envoy.yaml

Reply:
Okay I'll fork orchestro onto the BitRAW organization and there I'll add a BitRAW specific `envoy.yaml`.